src/main.py

```diff
@@ -9,7 +9,7 @@ def main():
     print("hello world from pyflag")
     print("here is my secret from env var:", SECRET)
 
-    print("I also have imported pip package discord.py see `discord.__dir__()`:", discord.__dir__())
+    print("I also have imported pip package discord.py see `discord.__dir__()`:", dir(discord))
```

Comment:
Is it related with this PR?

Reply:
If you read the comment it is asked by pylint, otherwise the check doesn't pass!

Reply:
Ok sorry, didn't read it! Perfect then :)